test_project/dbtvault_test/macros/schema_tests/tests.sql

```diff
@@ -30,23 +49,23 @@ expected_data AS (
     SELECT * FROM {{ ref(expected_seed) }}
 ),
 order_actual_data AS (
-    SELECT {{ source_columns_string }}
+    SELECT DISTINCT {{ source_columns_string }}
```

Comment:
Distinct should be removed here, as discussed. This causes problems for duplicate comparison.